control-plane/stor-port/src/types/v0/store/snapshots/replica.rs

```diff
@@ -117,6 +121,14 @@ impl ReplicaSnapshotMeta {
     pub fn size(&self) -> u64 {
         self.size
     }
+    /// Get the snapshot size.
```

Comment:
"/// Get the snapshot timestamp."

Reply:
Thanks, there were a few more too, fixed all I think.